pkg/cli/server/server.go

```diff
@@ -103,6 +106,10 @@ func run(app *cli.Context, cfg *cmds.Server) error {
 	serverConfig.ControlConfig.ClusterReset = cfg.ClusterReset
 	serverConfig.ControlConfig.EncryptSecrets = cfg.EncryptSecrets
 
+	if serverConfig.ControlConfig.SupervisorPort == 0 {
```

Comment:
do we assume its always going to be 0, because I cant see it in the list of server flags?

Reply:
RKE2 sets this port, for k3s it will always be zero.

Reply:
okay thanks for clarifying